### PR TITLE
Cross-build play-testkit to 2.11

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by raboof** [comment]
Not sure why the `03-cross-scala` test now fails?

**Comment by dwijnand** [comment]
I can't see Akka Jenkins, so I have no idea.

**Comment by raboof** [comment]
Oh that was intended to be public, I think I fixed that, could you doublecheck?

**Comment by dwijnand** [comment]
Yep, I can see it, and I understand why.  Will try and fix.

File: play-testkit/src/main/java/akka/grpc/play/JavaAkkaGrpcClientHelpers.java [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 package akka.grpc.play;
 
+import static scala.compat.java8.JFunction.*;
```
[the remark this answers is not in the document]
Comment (dwijnand) [reply play-testkit/src/main/java/akka/grpc/play/JavaAkkaGrpcClientHelpers.java]: ?

[the remark this answers is not in the document]
Comment (2m) [reply play-testkit/src/main/java/akka/grpc/play/JavaAkkaGrpcClientHelpers.java]: Me and raboof were wondering if helpers to create Java functions are available somewhere.

https://github.com/akka/akka-grpc/pull/414#discussion_r219888603

However I am not sure if there is a helpers to create `akka.japi.function.Function` instances.

**Pull request merged**